modules/service/src/main/scala/lucuma/odb/api/service/Init.scala

```diff
@@ -42,7 +38,89 @@ object Init {
     "dec": { "milliarcsecondsPerYear": 0.0 }
   },
   "radialVelocity": { "metersPerSecond": 423607 },
-  "parallax":       { "milliarcseconds":  0.00 }
+  "parallax":       { "milliarcseconds":  0.00 },
+  "sourceProfile": {
+    "point": {
+      "bandNormalized": {
+        "sed": {
+          "galaxy": "SPIRAL"
+        },
+        "brightnesses": [
+          {
+            "brightness": {
+              "value": 12.7,
+              "units": "VEGA_MAGNITUDE"
+            },
+            "band": "B"
+          },
+          {
+            "brightness": {
+              "value": 10.279,
+              "units": "VEGA_MAGNITUDE"
```

Comment:
On the input (i.e., here) I could turn this around

```
"brightnesses": [
  {
    "vega": 10.279,
    "band": "B"
  },
  ...
]
```

 to match the way it is done elsewhere (like for "parallax" above):

```
"parallax": { 
  "milliarcseconds": 1.0
}
```

I like that better and it is more consistent.  The problem is that on output (that is, when queried) I cannot support getting the value in any other units except those in which it was entered. I don't think we can easily translate between them.   Whereas for parallax we can cleverly provide the value in μas even though it was entered as mas:

```
"parallax": {
  "microarcseconds": 1000
}
```
I don't think we can do that for brightness. You couldn't take that `10.279` value and get the equivalent (if there is an equivalent) in, say, `erg/s/cm²/Hz`.   So the output would, inconsistently, continue to be "value" and "units".    :-/

Reply:
Right, some of the units can be converted but not every one. Also you can't convert surface to integrated or viceversa.